spec/annexes.html

```diff
@@ -125,7 +125,7 @@ <h1>Additions and Changes That Introduce Incompatibilities with Prior Editions</
       <emu-xref href="#sec-the-intl-collator-constructor"></emu-xref>, <emu-xref href="#sec-intl-numberformat-constructor"></emu-xref>, <emu-xref href="#sec-intl-datetimeformat-constructor"></emu-xref> In ECMA-402, 1st Edition, constructors could be used to create Intl objects from arbitrary objects. This is no longer possible in 2nd Edition.
     </li>
     <li>
-      <emu-xref href="#sec-intl.datetimeformat.prototype.format"></emu-xref> In ECMA-402, 1st Edition, the _length_ property of the function object _F_ was set to 0. In 2nd Edition, _length_ is set to 1.
+      <emu-xref href="#sec-intl.datetimeformat.prototype.format"></emu-xref> In ECMA-402, 1st Edition, the *"length"* property of the function object `F` was set to *0*. In 2nd Edition, *"length"* is set to *1*.
```

Comment:
I think `function object _F_` can stay as-is here, since it's a spec variable in the linked algorithm.